qemu/tests/cfg/timedrift_with_multi_vms.cfg

```diff
@@ -0,0 +1,24 @@
+- timedrift_with_multi_vms:
+    only Linux
+    type = timedrift_with_multi_vms
+    qemu_stop = off
+    image_snapshot = yes
+    start_vm = no
+    max_vms = 4
+    ntp_cmd = "(systemctl stop chronyd || service ntpdate stop)"
+    ntp_cmd += " && (chronyd -q 'server clock.redhat.com iburst'"
+    ntp_cmd +=  " || ntpdate clock.redhat.com)"
```

Comment:
better to separate the commands in `ntp_cmd` into functions. As mentioned above, please use `avocado.utils.service` to manage host service, which is `ntp` in your case.